File: app/src/main/java/org/stepic/droid/web/StepicProfileResponse.java
package org.stepic.droid.web;

import org.stepic.droid.model.Profile;
import org.stepic.droid.model.User;

import java.util.List;

public class StepicProfileResponse {
    private List<User> users;
    private List<Profile> profiles;

    public Profile getProfile() {
        if (profiles == null || profiles.size() < 1)
            return null;
        else
            return profiles.get(0);
    }
}

File: app/src/main/java/org/stepic/droid/web/StepicProfileResponse.kt
package org.stepic.droid.web

import org.stepik.android.model.user.Profile
import  org.stepik.android.model.user.User

class StepicProfileResponse(
        private val users: List<User>? = null,
        private val profiles: List<Profile>? = null
) {
    fun getProfile(): Profile? = profiles?.firstOrNull()
    fun getUser(): User? = users?.firstOrNull()
}
